<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlphaTestDialog</class>
 <widget class="QDialog" name="AlphaTestDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>208</width>
    <height>203</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Alpha test</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="selectionLayout">
     <item>
      <widget class="QRadioButton" name="neverRadioButton">
       <property name="text">
        <string>Never</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="lessRadioButton">
       <property name="text">
        <string>Less</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="lessOrEqualRadioButton">
       <property name="text">
        <string>Less or equal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="equalRadioButton">
       <property name="text">
        <string>Equal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="notEqualRadioButton">
       <property name="text">
        <string>Not equal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="greaterRadioButton">
       <property name="text">
        <string>Greater</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="greaterOrEqualRadioButton">
       <property name="text">
        <string>Greater or equal</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="alwaysRadioButton">
       <property name="text">
        <string>Always</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="valueSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="decimals">
      <number>4</number>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
